From ebea155605be1148f649e06862d7b387c4c5c691 Mon Sep 17 00:00:00 2001
From: Linu Cherian <lcherian@marvell.com>
Date: Wed, 28 Aug 2019 12:04:55 +0530
Subject: [PATCH 361/767] coresight: etm4x: Add Marvell OcteontTx CN9xxx ETM
 device

commit d51ad1e9deca6fde2b7bbf3fa7e48b9106ac4de4 from
git@git.assembla.com:cavium/WindRiver.linux.git

Add device id for Marvell OcteonTx CN9xxx ETM device.

Change-Id: If679a26626c5a2feb744af177cee438e3a1a26da
Signed-off-by: Linu Cherian <lcherian@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/16634
[Kevin: Use CS_AMBA_ID() instead of ETM4x_AMBA_ID()]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/hwtracing/coresight/coresight-etm4x.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/hwtracing/coresight/coresight-etm4x.c b/drivers/hwtracing/coresight/coresight-etm4x.c
index 8bb0092c7ec2..2a93036fa31f 100644
--- a/drivers/hwtracing/coresight/coresight-etm4x.c
+++ b/drivers/hwtracing/coresight/coresight-etm4x.c
@@ -1191,6 +1191,7 @@ static const struct amba_id etm4_ids[] = {
 	CS_AMBA_ID(0x000bb95a),		/* Cortex-A72 */
 	CS_AMBA_ID(0x000bb959),		/* Cortex-A73 */
 	CS_AMBA_UCI_ID(0x000bb9da, uci_id_etm4),	/* Cortex-A35 */
+	CS_AMBA_ID(0x000cc210),		/* Marvell-OcteonTx-CN9xxx */
 	{},
 };
 
-- 
2.31.1

